modules/axhal/src/platform/aarch64_common/boot_el2.rs

```diff
@@ -132,6 +132,9 @@ unsafe extern "C" fn _start() -> ! {
     // PC = 0x8_0000
     // X0 = dtb
     core::arch::asm!("
+        // save DTB pointer
```

Comment:
why moving this to the beginning of the code block.

Reply:
Prevent the address of dtb from being overwritten by subsequent operations like this：
https://github.com/arceos-hypervisor/arceos/blob/c62b386e829cc2252885b90b9d5d7aad9ead6a64/modules/axhal/src/platform/aarch64_common/boot_el2.rs#L141

Reply:
get it, thanks